clusters/build-clusters/01_cluster/machine_config/m5d4x_machineconfig.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         contents: |-
           [Mount]
           What=/dev/md/containerraid
-          Where=/var/lib/containers
+          Where=/var
```

Comment:
Why were we running container raid in thefirst place?

Reply:
The instance types have two attached NVMe devices.  CI workload is heavily container images - what else do you suggest?